frontend/test/lit/test_decomposition.py

```diff
@@ -70,9 +70,11 @@ def get_c_interface():
             """
             system_extension = ".dylib" if platform.system() == "Darwin" else ".so"
             lib_path = (
-                get_lib_path("runtime", "RUNTIME_LIB_DIR") + "/librtd_dummy" + system_extension
+                get_lib_path("runtime", "RUNTIME_LIB_DIR")
+                + "/librtd_null_device"
+                + system_extension
             )
-            return "dummy.remote", lib_path
+            return "NullDevice", lib_path
```

Comment:
It can be 'null.device'—there is nothing against it. I was just trying to normalize the names. In some places, we had CustomDevice, in other places `dummy.remote`. In the Lightning repo we have 'LightningKokkosSimulator' for LKokkos. I don't know the relevance of the naming here for Catalyst, so if you say it is better to make it `null.device` I can do that. Just let me know. :slightly_smiling_face:

Reply:
No, I just haven't seen this part of the code in a long time, so if you say it is more common to have another string, then let's keep your original name. Cheers! :)

Reply:
Yes I think `*.*` is the typical syntax for the device short name that users will use to construct it in Python. But here the provided name is used as a C++ class or function (the library entry point), so the dot might not even be legal?